src/jvm/clj_diff/FastStringOps.java

```diff
@@ -1,37 +0,0 @@
-package clj_diff;
-
-/**
- * Fast string operations for clj-diff.
- */
-public class FastStringOps {
```

Comment:
Removing a class would also be a breaking change.

Reply:
yes, it is thus the new version indicates API change `2.0.0-experimental`